# Engine id
string id
---
# Error string (empty string if no error)
string error
